include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -1008,10 +1037,12 @@ FEPointEvaluation<n_components, dim, spacedim, Number>::evaluate(
 
       const auto unit_points =
         mapping_info->get_unit_points(current_cell_index, current_face_number);
+      const auto &mapping_data =
+        mapping_info->get_mapping_data(current_cell_index, current_face_number);
```

Comment:
:+1: But the compiler should have done this for you.

Reply:
He hasn't ... now it is definitely fixed!

Reply:
I guess this was some aliasing that prevented the compiler? A reference should otherwise not have a big effect, because it only avoids following the pointers, not the data access.